### PR TITLE
Remove $repoTag image after every successful push

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

**Comment by nghiant2710** [comment]
@Page- please review this PR

File: bashbrew/bashbrew.sh [hunk 1]
```diff
@@ -413,6 +414,7 @@ while [ "$#" -gt 0 ]; do
 						"$docker" rmi -f "$namespace/$repoTag-$dateStamp"
```
Comment (Page-) [inline bashbrew/bashbrew.sh]: You also need to remove the non-namespaced version here right?

Comment (nghiant2710) [inline bashbrew/bashbrew.sh]: it's added below in this PR

Comment (Page-) [inline bashbrew/bashbrew.sh]: That removes `$repoTag`, what about `$repoTag-$dateStamp`? Or do we not create a datestamped version without the namespace?

Comment (nghiant2710) [inline bashbrew/bashbrew.sh]: Yeah, we don't create datestamped version without namespace so only need to remove $repoTag

**Comment by Page-** [comment]
LGTM

**Pull request merged**